src/Altinn.Platform/Altinn.Platform.Storage/DataCleanup/Altinn.Platform.Storage.DataCleanup.csproj

```diff
@@ -6,11 +6,11 @@
     <ProjectGuid>{3F69C9C9-D602-490C-B2BA-A5FB27E363EE}</ProjectGuid>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Altinn.Platform.Storage.Interface" Version="2.5.10" />
+    <PackageReference Include="Altinn.Platform.Storage.Interface" Version="3.0.1" />
     <PackageReference Include="Azure.Identity" Version="1.3.0" />
     <PackageReference Include="Azure.Security.KeyVault.Secrets" Version="4.1.0" />
     <!--12.5.1 breaks authorization towards blob storage. Issue: #4650. -->
-    <PackageReference Include="Azure.Storage.Blobs" Version="12.4.4" />
+    <PackageReference Include="Azure.Storage.Blobs" Version="12.8.1" />
```

Comment:
Note the comment above this line. It should be testes before the package is bumped. please do a manual test or revert this change